fvm/src/call_manager/mod.rs

```diff
@@ -111,6 +119,15 @@ pub trait CallManager: 'static {
         delegated_address: Option<Address>,
     ) -> Result<()>;
 
+    // returns the actor call stack
+    fn get_actor_call_stack(&self) -> &Vec<(ActorID, &'static str)>;
```

Comment:
```suggestion
    fn get_actor_call_stack(&self) -> &[(ActorID, &'static str)];
```

(slightly more idiomatic)